tests/test_sql_query.py

```diff
@@ -34,6 +34,7 @@ class TestSQL(tb.SQLQueryTestCase):
     SETUP = os.path.join(
         os.path.dirname(__file__), 'schemas', 'movies_setup.edgeql'
     )
+    TRANSACTION_ISOLATION = False
```

Comment:
Add a comment saying why isolation is off.

Reply:
Actually it might be cleaner to add the extra objects being created to the schema for the whole test suite.

Otherwise you'll also want to modify that test to clean up the things created also